portality/app.py

```diff
@@ -168,6 +168,16 @@ def utc_timestamp(stamp, string_format="%Y-%m-%dT%H:%M:%SZ"):
     utcdt = datetime(tt.tm_year, tt.tm_mon, tt.tm_mday, tt.tm_hour, tt.tm_min, tt.tm_sec, tzinfo=pytz.utc)
     return utcdt.strftime(string_format)
 
+@app.template_filter('doi_url')
+def doi_url(doi):
+    """
+    Create a link from a DOI.
+    :param doi: the string DOI
+    :return: the HTML link
+    """
+    tendot = doi[doi.find('10.'):]
+    return "<a href='http://dx.doi.org/{0}'>{0}</a>".format(tendot)
```

Comment:
Why are we stripping the 10.? I assume this works as you've committed it but I've never seen people explicitly remove the part of the handle that identifies it as a DOI handle...